package.json

```diff
@@ -51,8 +51,8 @@
   },
   "dependencies": {
     "@nomiclabs/hardhat-etherscan": "^3.0.3",
-    "@openzeppelin/contracts": "^4.7.3",
-    "@openzeppelin/contracts-upgradeable": "^4.8.2",
+    "@openzeppelin/contracts": "^5.0.1",
```

Comment:
how do we keep old version of contract?

Reply:
My bad. I've put you in review before the PR is finalised.
Now I've added two versions of OpenZeppelin.